Dockerfile

```diff
@@ -1,13 +1,17 @@
-FROM debian:wheezy
+FROM debian:stretch
 
 RUN apt-get update && \
     apt-get upgrade -y && \
-    apt-get install -y build-essential curl libssl-dev && \
+    apt-get install -y build-essential wget libssl-dev && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-RUN curl https://www.unrealircd.org/downloads/Unreal3.2.10.4.tar.gz | tar xz && \
-    cd Unreal3.2.10.4 && \
+RUN gpg --keyserver pgp.mit.edu --recv-keys 0xA7A21B0A108FF4A9 && \
+    wget https://www.unrealircd.org/unrealircd4/unrealircd-4.0.12.1.tar.gz.asc && \
```

Comment:
Shouldn't we have this locally somewhere to verify? If they are compromised it's pretty safe to assume that this `asc` will be compromised as well.